src/dict.h

```diff
@@ -83,6 +86,7 @@ typedef struct dictType {
     unsigned int keys_are_odd : 1;
     /* TODO: Add a 'keys_are_even' flag and use a similar optimization if that
```

Comment:
It doesn't exist. When I added `keys_are_odd`, Oran wanted me to add a TODO for this, so I added this comment, but it's not possible to use the same trick as we do for odd keys (as we do now, using LSB=1 to say it's a key instead of a dictEntry pointer).

Reply:
I always thought we should remove the keys_are_even TODO, but 🤷

Reply:
Me too. Let's remove the todo. :)